go/vt/vtgate/executor.go

```diff
@@ -216,6 +216,7 @@ func (e *Executor) Execute(ctx context.Context, method string, safeSession *Safe
 
 	logStats.SaveEndTime()
 	QueryLogger.Send(logStats)
+	TimingStatistics.RecordStats(logStats)
```

Comment:
is this different from what we produce here https://github.com/vitessio/vitess/blob/main/go/vt/vtgate/vtgate.go#L205

Reply:
stats.MultiTimings is a histogram with multi labels so you should be able to get p0 to p100 range.

3 metrics are generated with it

```go
	_ = stats.NewRates("QPSByOperation", stats.CounterForDimension(rpcVTGate.timings, "Operation"), 15, 1*time.Minute)
	_ = stats.NewRates("QPSByKeyspace", stats.CounterForDimension(rpcVTGate.timings, "Keyspace"), 15, 1*time.Minute)
	_ = stats.NewRates("QPSByDbType", stats.CounterForDimension(rpcVTGate.timings, "DbType"), 15*60/5, 5*time.Second)
```

Reply:
```
		timings: stats.NewMultiTimings(
			"VtgateApi",
			"VtgateApi timings",
			[]string{"Operation", "Keyspace", "DbType"}),
```

It is being recorded on Execute methods like 

```go
	statsKey := []string{"Execute", destKeyspace, topoproto.TabletTypeLString(destTabletType)}
	defer vtg.timings.Record(statsKey, time.Now())
```

Reply:
hey, thanks for the review. True, there are currently histograms here inside stats package, but they only track buckets rather than percentiles, unless I missed something?

Reply:
For an example of how to get p95/p99 metrics from the histograms, see https://github.com/vitessio/vitess/blob/main/vitess-mixin/dashboards/resources/config/vtgate_config.libsonnet#L404

This example is for Grafana using prometheus. Not sure what you use for your metrics, but similar functions should be available.

Reply:
sounds good. But here we also emit the `totalQueryTime` and some mysql connection metrics such as the `mysqlQueryTime` and `connectionAcquisitiontime` and I don't believe we have anything currently that does that?